docs/core/whats-new/dotnet-core-3-0.md

```diff
@@ -309,6 +343,20 @@ The [Windows Application Packaging Project](https://docs.microsoft.com/windows/u
 <RuntimeIdentifiers>win-x86;win-x64</RuntimeIdentifiers>
 ```
 
+## WinForms HighDPI
+
+.NET Core Windows Forms applications can set High DPI mode with <xref:System.Windows.Forms.Application.SetHighDpiMode(System.Windows.Forms.HighDpiMode)>. The `SetHighDpiMode` method will set the corresponding High DPI mode unless the setting has been set by other means like `App.Manifest` or P/Invoke before `Application.Run`.
+
+The possible `HighDPIMode` values, as expressed by the `HighDpiMode` enum are:
```

Comment:
xref links here to the enum and the fields? And what are these `HighDPIMode` values, that is, what's that first one referring to?

Reply:
I cant xref to individual items, but I can add one to the enum.